pyannote/audio/tasks/segmentation/multilabel_detection.py

```diff
@@ -78,19 +80,19 @@ class SpeakerTracking(SegmentationTaskMixin, Task):
     """
 
     def __init__(
-        self,
-        protocol: Protocol,
-        duration: float = 2.0,
-        warm_up: Union[float, Tuple[float, float]] = 0.0,
-        balance: Text = None,
-        weight: Text = None,
-        batch_size: int = 32,
-        num_workers: int = None,
-        pin_memory: bool = False,
-        augmentation: BaseWaveformTransform = None,
-        metric: Union[Metric, Sequence[Metric], Dict[str, Metric]] = None,
+            self,
+            protocol: Protocol,
+            classes: Optional[List[str]] = None,
+            duration: float = 2.0,
+            warm_up: Union[float, Tuple[float, float]] = 0.0,
+            balance: Text = None,
+            weight: Text = None,
+            batch_size: int = 32,
+            num_workers: int = None,
+            pin_memory: bool = False,
+            augmentation: BaseWaveformTransform = None,
+            metric: Union[Metric, Sequence[Metric], Dict[str, Metric]] = None,
```

Comment:
I haven't thought a lot about the validation metric for this type of task.
It relies on AUROC right? But what does it mean for multilabel classification?